### ISSUE TITLE
[Issue] Two files when use a dot in filename

### ISSUE THREAD

**Issue by gtrabanco** [YsarocK/fetch-appwrite-types#59]
I wanted to generate a file with extension `.d.ts` instead of `.ts` so I changed the command to:

```shell
npx fetch-appwrite-types outDir ./types outFileName appwrite.d hardTypes
```

And it generated two files:

```
ls -1 types
appwrite.d.ts
appwrite.ts
```

### `appwrite.ts`

Contains all enum types and `Email`, `URL` types.

### `appwrite.d.ts`

Contains all collections types. The types miss all enums in `appwrite.ts` file and, of course, the Email and URL types.

## Other

Looks like the issue is here:
- https://github.com/YsarocK/fetch-appwrite-types/blob/1d0993d5baac2cf9b8469d016d555d2bf38ad863/src/main.ts#L38
- https://github.com/YsarocK/fetch-appwrite-types/blob/1d0993d5baac2cf9b8469d016d555d2bf38ad863/src/utils/CreateHardFieldsTypes.ts#L5

CreateHardFieldsType has hardcoded filename.

## Non related

There should be an option to create camelCase or upper names for enums.

**Comment by YsarocK** [comment]
Hi @gtrabanco 

You're right, will give a look at it this week!

Thanks for reporting this

**Comment by YsarocK** [comment]
Hi @gtrabanco,

Should be fixed in 1.8.3!
